dashboard/db/schema.rb

```diff
@@ -10,7 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
+ActiveRecord::Schema.define(version: 2024_12_04_175716) do
+=======
 ActiveRecord::Schema.define(version: 2024_12_03_163649) do
+>>>>>>> staging-next
```

Comment:
Sorry, I might not have been clear ... you should resolve the conflict but I think accepting your change here is fine.

Reply:
Oh whoops that's odd? I did resolve it and clicked "Accept" in the VSCode display, but then it kept the `<<<< HEAD` lines? I'll fix that!